pkg/simulation/run.go

```diff
@@ -126,9 +126,9 @@ func (s *Simulation) AdvanceFrame() error {
 		delay = s.C.AnimationCancelDelay(act.Typ, act.Param) + s.C.UserCustomDelay()
 		//check if we should delay
 
-		//so if current frame - when the last action is used is >= delay, then we shouldn't
+		//so if current frame - when the last action is used is > delay, then we shouldn't
 		//delay at all
-		if s.C.F-s.lastActionUsedAt >= delay {
+		if s.C.F-s.lastActionUsedAt > delay {
```

Comment:
So did you remember what this change was needed for?

Reply:
yes it is in the commit message `Fix sim treating 1f delay as no delay`.